.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+pom.xml @andrewazores @ebaron
+src/container/ @andrewazores @ebaron
```

Comment:
This would be `src/main/docker` in the new repo

Reply:
Now that we're using org teams too, this could be `@cryostatio/maintainers` like Thuan did here: https://github.com/cryostatio/cryostat-operator/pull/648/files